test/router.js

```diff
@@ -35,6 +35,21 @@ describe('Router', function () {
     router({}, {}, done)
   })
 
+  describe('.removeAllRoutes()', function () {
+    it('should remove all routes from the stack', function (done) {
+      var router = new Router()
+
+      for (var i = 0; i < 10; i++) {
+        router.get('/thing' + i, helloWorld)
+      }
+
+      router.removeAllRoutes()
+
+      assert.equal(router.stack.length, 0)
```

Comment:
Tests should only be asserting against the public-visible behavior/API. I assume that by adding this method that you are saying that `router.stack` is private and not public. This test using `.stack` to make an assertion would seemingly re-enforce the idea that it is public (as changing the name of the property would break this test -- the indicator it's not private). If it is public, then I'm missing the purpose of this function, as `router.stack = []` would already be clearly possible for a user to do, is that right?

Reply:
Yes I could do that already, but having it in the router as a function makes it more "safe" against updates that could break that